frontend/src/scenes/insights/filters/ActionFilter/entityFilterLogic.ts

```diff
@@ -285,7 +285,7 @@ export const entityFilterLogic = kea<entityFilterLogicType>([
                     _filter.order = _filter.order + 1
                 }
             }
```

Comment:
If you're adding it under, then you need to change the logic here as well, you don't need to move all of the other ones "down" by increasing their order